exporters/etw/include/opentelemetry/exporters/etw/etw_tracer.h

```diff
@@ -899,7 +899,15 @@ class Span : public opentelemetry::trace::Span
    */
   void End(const opentelemetry::trace::EndSpanOptions &options = {}) noexcept override
   {
-    end_time_ = std::chrono::system_clock::now();
+    if (options.end_steady_time == opentelemetry::common::SteadyTimestamp())
```

Comment:
Update this to compare with 0 explicitly. And add TODO to choose a different value to represent "not set" status, as 0 is a valid value, and the user might set endtime to it.

Reply:
Thanks fixed this.